README.md

```diff
@@ -4,9 +4,8 @@ This handy feature will bring up an entire Screwdriver instance (ui, api, and lo
 for you to play with.  All data written to a database will be stored in `data` directory.
 
 Requires:
- - Python 2.7
- - Mac OSX 10.10+
- - [Docker for Mac][docker]
+ - Python 2.7.11+
```

Comment:
Python 2.7.11+ is needed because that is the first release that has support for TLS 1.2 in the Python standard library.